psh/wget/wget.c

```diff
@@ -0,0 +1,491 @@
+/*
+ * Phoenix-RTOS
+ *
+ * wget - downloads a file using HTTP
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Maciej Purski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <ctype.h>
+#include <errno.h>
+#include <netdb.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <unistd.h>
+#include <arpa/inet.h>
+#include <sys/socket.h>
+#include <sys/minmax.h>
+#include <sys/time.h>
+
+#include "../psh.h"
+
+
+static struct {
+	char buf[4096];
+	char *readptr;
+	int len;
+	FILE *outfile;
+	const char *path;
+	char *host;
+	const char *filename;
+	int debug;
+	int socket;
+} common;
+
+
+void psh_wgetinfo(void)
+{
+	printf("downloads a file using http");
+}
+
+
+static void wget_help(void)
+{
+	printf("Usage: wget [options] ... URL\n");
+	printf("Options\n");
+	printf("  -h:  prints help\n");
+	printf("  -O:  output file\n");
```

Comment:
I missed that one somehow, but it may be multi-line string, that saves three unnecessary `printf` calls.
See the difference with the Godbolt's tool (https://gcc.godbolt.org/z/s7arW8Gnh).
```suggestion
	printf("Usage: wget [options] ... URL\n"
		   "Options\n"
		   "  -h:  prints help\n"
		   "  -O:  output file\n");
```